docs/deploying/maincloud.md

```diff
@@ -0,0 +1,29 @@
+# SpacetimeDB Cloud Deployment
+
+The SpacetimeDB Cloud is a managed cloud service that provides developers an easy way to deploy their SpacetimeDB apps to the cloud.
+
+## Deploy via CLI
+
+1. Install the SpacetimeDB CLI and create your module (see [Getting Started](/docs/getting-started))
+1. Publish to maincloud:
+
+```bash
+spacetime publish -s maincloud my-cool-module
+```
+
+## Connecting your Identity to the Web Dashboard
```

Comment:
I would document (maybe Tyler can comment as well):

1. If I deploy a module with some name `my-example-module` how do I bring this module up on the dashboard?
2. Is there any way in the dashboard to check to see if the module is running or suspended? How do you check this? How do you pause the module?
3. At the bottom of this document should we have like a "Next Step: explore the dashboard" or is there something we should recommend that users look at next?

Reply:
If someone else can add this content (like Tyler), they're definitely welcome to. I don't have this content in my knowledge centers post-great-web-rework, though. I can add the link if that's desirable.

Reply:
@jdetter how about we create a followup ticket for this for Tyler's input?

Reply:
Created follow-up ticket #201 .

Reply:
1. If you deploy a module with some name `my-example-module` you can either:

- Navigate to your profile where you'll see a list of all of your modules. You can go to your profile at any of `/`, `/profile` `/@your-username`.

or

- Navigate to `/my-example-module` directly.

2. There is no way to see if a module is running or suspended, however it should be noted that going to the dashboard will connect a user and thus will start it running if it wasn't. There is currently no way to pause module execution, although I have it on my list of nice-to-haves for the website. Will do asap after the release I think.

3. Definitely that would be good to do, although I think that should maybe come later.